docs/debug-mode.rst

```diff
@@ -26,3 +26,47 @@ The debugging context can already check for:
 An HPy module may be loaded in debug mode using::
 
   mod = hpy.universal.load(module_name, so_filename, debug=True)
+
+The default module loader generated by HPy also recognizes environment
+variable ``HPY_DEBUG_CONTEXT``. If ``HPY_DEBUG_CONTEXT=1``, then all HPy
+modules are loaded with debug context. Alternatively ``HPY_DEBUG_CONTEXT``
+can be set to a comma separated list of names of the modules that should
+be loaded in the debug mode.
+
+The HPy debug module also exposes ``LeakDetector`` class for detection of
+leaked handles. ``LeakDetector`` can be used to check that some code does
+not leave behind unclosed ``HPy`` handles. For example:
+
+.. literalinclude:: examples/tests.py
+  :language: python
+  :start-at: def test_leak_detector
+  :end-at: # Run some HPy extension code
+
+Additionally, the debug module also exposes pytest fixture ``hpy_debug`` that
+for the time being enables the ``LeakDetector``, but may also enable other
+useful debugging facilities.
+
+.. literalinclude:: examples/tests.py
+  :language: python
+  :start-at: from hpy.debug.pytest import hpy_debug
+  :end-at: # Run some HPy extension code
+
+**ATTENTION**: the usage of ``LeakDetector`` or ``hpy_debug`` by itself does not
+enable the HPy debug mode! If the debug mode is not enabled for any extension,
+then those features do nothing useful (but also nothing harmful).
+
+When dealing with handle leaks, it is useful to get a stack trace of the
+allocation of the leaked handle. This feature has large memory requirements
+and is therefore opt-in. It can be activated by:
+
+.. literalinclude:: examples/tests.py
+  :language: python
+  :start-at: hpy.debug.set_handle_stack_trace_limit
+  :end-at: hpy.debug.set_handle_stack_trace_limit
+
+and disabled by:
+
+.. literalinclude:: examples/tests.py
+  :language: python
+  :start-at: hpy.debug.disable_handle_stack_traces
+  :end-at: hpy.debug.disable_handle_stack_traces
```

Comment:
Yes, you're right. I am using `add2line`, which seem to work w/o `-rdynamic`. Maybe the `__repr__` that prints the stack trace could also include a link to the docs where we'd document all those tricks.

Reply:
ah, so you get the "raw" stacktrace and convert it manually after it's printed?
This also works, although it would be nice to have it automatically of course.
+1 to add a link to some docs/howto